apps/docs/content/guides/platform/compute-add-ons.mdx

```diff
@@ -4,44 +4,40 @@ title: 'Compute Add-ons'
-| Micro        | $0.01344         | ~$10              | 2-core ARM (shared)     | 1 GB         | 60                  | 200                 | 10GB                        |
-| Small        | $0.0206          | ~$15              | 2-core ARM (shared)     | 2 GB         | 90                  | 400                 | 50GB                        |
-| Medium       | $0.0822          | ~$60              | 2-core ARM (shared)     | 4 GB         | 120                 | 600                 | 100GB                       |
-| Large        | $0.1517          | ~$110             | 2-core ARM (dedicated)  | 8 GB         | 160                 | 800                 | 200GB                       |
-| XL           | $0.2877          | ~$210             | 4-core ARM (dedicated)  | 16 GB        | 240                 | 1,000               | 500GB                       |
-| 2XL          | $0.562           | ~$410             | 8-core ARM (dedicated)  | 32 GB        | 380                 | 1,500               | 1TB                         |
-| 4XL          | $1.32            | ~$960             | 16-core ARM (dedicated) | 64 GB        | 480                 | 3,000               | 2TB                         |
-| 8XL          | $2.562           | ~$1,870           | 32-core ARM (dedicated) | 128 GB       | 490                 | 6,000               | 4TB                         |
-| 12XL         | $3.836           | ~$2,800           | 48-core ARM (dedicated) | 192 GB       | 500                 | 9,000               | 6TB                         |
-| 16XL         | $5.12            | ~$3,730           | 64-core ARM (dedicated) | 256 GB       | 500                 | 12,000              | 10TB                        |
-
-Number of connections above are recommended values.
+We charge hourly for additional compute based on your usage. Read more about [usage-based billing for compute](/docs/guides/platform/org-based-billing#billing-for-compute-compute-hours).
 
-<Admonition type="note" label="Recommended Database Size">
+<Admonition type="note" label="Free Upgrade to Micro Instance for Paying Project">
 
-The database sizes recommended for each add-on are intended purely as a guide and recommendation to ensure performance is optimal, these sizes are not enforced.
+All paying projects have free upgrades from Nano to Micro instances.
 
-The performance of your database is impacted by many factors, some of them being the resources available to it or the size of the data contained within it. See the [shared responsibility model](https://supabase.com/docs/guides/platform/shared-responsibility-model) for more information.
+We recommend that you take advantage of the free upgrade when your usage requires Micro instance resources and you can [incur some downtime](/docs/guides/platform/compute-add-ons#upgrade-downtime).
```

Comment:
Think this should mention that the minimum compute costs for _any_ instance, even nano, on paid plans is $10. So there is no advantage in staying on nano for a paying customer.

Reply:
alright, i pushed up another commit that should address this.